### PR TITLE
YLW 3119 -  Create book and library classes

### PR DESCRIPTION
**Pull request by davidgreenland**
#### What
Creates a Book and Library class

#### Why
To define the properties of a book an to be able to able to add them to a Library catalogue.

#### How
- Created a book class with Author, Title and Spell properties
- Added a static field that keeps track of number of instances of Book class to use as an ID property.
- Printed some test Catalogues to the console

### CODE REVIEW

File: TheEnchantedLibrary/Models/Book.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+﻿namespace TheEnchantedLibrary.Models;
+
+public class Book
+{
+    public static int CountOfBooks { get; private set; }
```
Comment (SiddiqueAbdullah) [inline TheEnchantedLibrary/Models/Book.cs]: no

**Review by SiddiqueAbdullah — CHANGES_REQUESTED** [review]

File: TheEnchantedLibrary/Models/Library.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+﻿namespace TheEnchantedLibrary.Models;
+
+public class Library
+{
+    public ICollection<Book> Catalogue { get; private set; } = [];
```
Comment (SiddiqueAbdullah) [inline TheEnchantedLibrary/Models/Library.cs]: no property

File: TheEnchantedLibrary/Models/Book.cs (hunk 1 above)
Comment (davidgreenland) [reply TheEnchantedLibrary/Models/Book.cs]: removed

File: TheEnchantedLibrary/Models/Library.cs (hunk 2 above)
Comment (davidgreenland) [reply TheEnchantedLibrary/Models/Library.cs]: done

**Review by SiddiqueAbdullah — APPROVED** [review]

**Pull request merged**